common-rendering/src/components/keyEvents.stories.tsx

```diff
@@ -1,58 +1,72 @@
 // ----- Imports ----- //
-
+import { css } from "@emotion/react";
 import { KeyEvent } from "./keyEvents";
 import KeyEvents from "./keyEvents";
 import { Pillar, Special, Theme } from "@guardian/types";
-import { css } from "@emotion/react";
 
 // ----- Stories ----- //
 
+const now = new Date();
+
+function addMinutes(date: Date, minutes: number) {
+	return new Date(date.getTime() - minutes * 60000).toISOString();
+}
+
+const currentTime = addMinutes(now, 0);
+const thirtySeconds = addMinutes(now, 0.5);
+const fiveMinutes = addMinutes(now, 5);
+const fifteenMinutes = addMinutes(now, 15);
+const oneHour = addMinutes(now, 60);
+const oneDay = addMinutes(now, 60 * 24);
+const oneWeek = addMinutes(now, 60 * 24 * 7);
+const oneYear = addMinutes(now, 60 * 24 * 7 * 52);
```

Comment:
I like that we're including all the different variants here as part of the key events !

Reply:
Yeah, this is really nice!

Reply:
Nice from me too 🙌